h2o-docs/src/product/data-science/gbm.rst

```diff
@@ -326,7 +316,7 @@ FAQ
 
 -  **What if there are a large number of columns?**
 
-  DRF models are best for datasets with fewer than a few thousand columns.
+  GBM models are best for datasets with a large number of columns (several thousand or more). DRF models are best for datasets with fewer than a few thousand columns.
```

Comment:
not true - all tree methods are getting less useful with growing number of columns, DRF and GBM are the same in that respect